meta-balena-common/recipes-containers/balena-supervisor/balena-supervisor/balena-supervisor.service

```diff
@@ -28,7 +28,6 @@ SyslogIdentifier=balena-supervisor
 EnvironmentFile=/etc/balena-supervisor/supervisor.conf
 EnvironmentFile=-/tmp/update-supervisor.conf
 ExecStartPre=-@BINDIR@/balena stop resin_supervisor
-ExecStartPre=-@BINDIR@/balena rm --force resin_supervisor
```

Comment:
Handle the container removal in `start-balena-supervisor` in case we need to get the supervisor image id from the existing containers.